packages/pangea-sdk/pangea/asyncio/request.py

```diff
@@ -183,7 +182,20 @@ async def put_presigned_url(self, url: str, files: Sequence[Tuple]):
         if resp.status < 200 or resp.status >= 300:
             raise pe.PresignedUploadError(f"presigned PUT failure: {resp.status}", await resp.text())
 
-    async def download_file(self, url: str, filename: Optional[str] = None) -> AttachedFile:
+    async def download_file(self, url: str, filename: str | None = None) -> AttachedFile:
+        """
+        Download file
+
+        Downloads a file from the specified URL and saves it with the given
```

Comment:
Nit, maybe make it imperative, like:

> Download a file from the specified URL and save it with the given filename.

I checked a few SDK docs, and it seems we have a mix, so no consistency between imperative and incomplete third-person singular, so I don't see a reason to keep the latter going forward - unless you think otherwise. 

In any case, this is minor and can apply to future updates.

Reply:
Done.